examples/framework.py

```diff
@@ -43,9 +45,16 @@ def __init__(self):
         self.argparse = argparse.ArgumentParser()
 
     def prepare_argparse(self):
+
+        config_default_path = os.path.join(os.environ.get("XDG_CONFIG_HOME",
+                                           os.path.expanduser("~/.config")),
+                                           "aioxmpp_examples.ini")
```

Comment:
Break the line between ``(`` and ``os.environ`` please, that will allow to align the default for ``.get`` with the opening parenthesis, making the line easier to read.